docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+if [ "$(id -u)" = '0' ]; then
+  chown -R dce:dce /out
+  exec su-exec dce "$0" "$@"
+fi
```

Comment:
Would be nice to have a comment explaining what this block does. I'm not a Linux user, so it's not immediately obvious to me.

Reply:
Ok I'll add some comments.